instruqt-tracks/network-infrastructure-automation/08-install-consul-terraform-sync/solve-workstation

```diff
@@ -28,12 +67,16 @@ while /bin/true; do
     app=$(curl -s -o /dev/null -w "%{http_code}" http://localhost:8080)
     if [ "${app}" != "200" ]; then
       echo "App did not return a 200, returned ${app}. Sleeping and retrying."
-      sleep 20
+
+        if [ "${app}" == "504" ]; then
+            /root/terraform/panos_commit/panos-commit -config /root/terraform/panos_commit/panos-commit.json -force
+            sleep 20
+            break
+        fi
```

Comment:
We can move the existing sleep to after the `if` check rather than completely deleting it
```suggestion
        if [ "${app}" == "504" ]; then
            /root/terraform/panos_commit/panos-commit -config /root/terraform/panos_commit/panos-commit.json -force
            sleep 20
            break
        fi
        sleep 20
```